<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>processingpopup</class>
 <widget class="QDialog" name="processingpopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#processingpopup {
background-color: rgb(39, 39, 39);
border: 1px solid white;
}</string>
  </property>
  <widget class="QLabel" name="statusLbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>35</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Processing footage...</string>
   </property>
  </widget>
  <widget class="QLabel" name="videoPreview">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>160</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: blue;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>58</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fileName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>83</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #888;</string>
   </property>
   <property name="text">
    <string>IMG_2004.MOV</string>
   </property>
  </widget>
  <widget class="QLabel" name="thumbnailCheckLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>47</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>âœ…</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
